graphbolt/src/csc_sampling_graph.cc

```diff
@@ -215,7 +222,7 @@ CSCSamplingGraph::BuildGraphFromSharedMemoryTensors(
   auto& optional_tensors = std::get<2>(shared_memory_tensors);
   auto graph = c10::make_intrusive<CSCSamplingGraph>(
       optional_tensors[0].value(), optional_tensors[1].value(),
-      optional_tensors[2], optional_tensors[3]);
+      optional_tensors[2], optional_tensors[3], torch::nullopt);
```

Comment:
@czkkkkkk Can you double check this can be supported by shared memory in the future PR?

Reply:
As we discussed, we will deal with it in later PRs.